src/backend/src/services/projects.services.ts

```diff
@@ -605,6 +606,58 @@ export default class ProjectsService {
   }
 
   /**
+   * Create an assembly
+   * @param name The name of the assembly to be created
+   * @param pdmFileName The name of the file holding the assembly
```

Comment:
you should rearrange these params to match the order below and specify that it's an optional field in the comment